android/src/main/java/com/thebluealliance/androidclient/activities/RecentNotificationsActivity.java

```diff
@@ -33,7 +41,7 @@ public void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         setupActionBar();
 
-        Fragment fragment = new RecentNotificationsFragment();
+        Fragment fragment = new RecentNotificationsFragmentV2();
```

Comment:
If the fragment is good, rename to `RecentNotificationsFragment` and remove the old one.